test/e2e/support/gutenberg-commands.js

```diff
@@ -4,7 +4,7 @@ Cypress.Commands.add( 'newPost', () => {
 
 Cypress.Commands.add( 'visitAdmin', ( adminPath ) => {
 	cy.visit( '/wp-admin/' + adminPath ).location( 'pathname' ).then( ( path ) => {
-		if ( path === '/wp-login.php' ) {
+		if ( /\/wp-login\.php$/.test( path ) ) {
```

Comment:
You could use also [endsWith](https://developer.mozilla.org/pl/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith) to make it even simpler. Not sure if that matters that much though :)

Reply:
Thank you. I will do that.

Reply:
@jaswrks @gziolo Actually, I don't think so, at least not in this way. See https://github.com/WordPress/gutenberg/pull/4945/files#r166864232.